playground/AzureServiceBus/ServiceBusWorker/Program.cs

```diff
@@ -1,10 +1,26 @@
+using Azure.Messaging.ServiceBus;
 using ServiceBusWorker;
 
 var builder = Host.CreateApplicationBuilder(args);
 
 builder.AddServiceDefaults();
 
-builder.AddAzureServiceBusClient("sbemulator");
+builder.AddAzureServiceBusClient("queueOne");
+
+builder.Services.AddSingleton(sp =>
+{
+    var client = sp.GetRequiredService<ServiceBusClient>();
+    return client.CreateSender("queue1");
+});
+builder.Services.AddSingleton(sp =>
+{
+    var client = sp.GetRequiredService<ServiceBusClient>();
+    return client.CreateProcessor("queue1", new ServiceBusProcessorOptions
+    {
+        MaxConcurrentCalls = 1, // Process one message at a time
+        AutoCompleteMessages = true
+    });
+});
```

Comment:
What's this jank?

Reply:
Since we're not doing a client integration to inject the Processors/Senders/Receivers, this showcases how someone could do that with the child resource connection string and existing DI APIs.

Reply:
😢

Reply:
future?

Reply:
Maybe?

One thing that makes Service Bus unique is that the child resources on the hosting side represent data entities (queues, topics, subscriptions) but the "child" concept on the client side represents actors like senders receivers this means that for any given child resource client integration there are two distinct options, one for the service bus and one for the processor receiver, etc.